### DISCUSSION TITLE
[NOTICE] System DNS configuration not usable yet, exceptionally resolving [...] using bootstrap resolvers over tcp

### DISCUSSION THREAD

**Discussion by MojtabaFaal** [DNSCrypt/dnscrypt-proxy#1828] [Q&A] [answered]
Hi
What is this **NOTICE**?

`[2021-08-17 15:04:00] [NOTICE] System DNS configuration not usable yet, exceptionally resolving [dns1.dnscrypt.ca] using bootstrap resolvers over tcp`

`[2021-08-17 15:04:12] [NOTICE] System DNS configuration not usable yet, exceptionally resolving [jp.tiarap.org] using bootstrap resolvers over tcp`

`[2021-08-17 15:05:04] [NOTICE] System DNS configuration not usable yet, exceptionally resolving [doh-2.seby.io] using bootstrap resolvers over tcp`

`[2021-08-17 15:07:19] [NOTICE] System DNS configuration not usable yet, exceptionally resolving [dns2.dnscrypt.ca] using bootstrap resolvers over tcp`

I receive this NOTICE **after Release 2.1.0**
What should I do ?

**Comment by MojtabaFaal** [comment, marked as answer]
@jedisct1

**Reply by jedisct1** [reply]
> Hi
> What is this **NOTICE**?
>
> What should I do ?

Nothing, just ignore the message. This is not a warning nor an error. You can set `ignore_system_dns = true`, if it's currently set to `false`, but this is not necessary.

**Reply by MojtabaFaal** [reply]
I receive this NOTICE after **Release 2.1.0**
I did not see this NOTICE in the previous version at all

**Reply by MojtabaFaal** [reply]
I did exactly what you said

`ignore_system_dns = true`

but I receive this NOTICE

`[2021-08-27 20:19:25] [NOTICE] System DNS configuration not usable yet, exceptionally resolving [doh.seby.io] using bootstrap resolvers over tcp`

Why ?

**Reply by MojtabaFaal** [reply]
In my opinion, the errors ( [NOTICE] , [CRITICAL] , [WARNING] ) in the new version have increased

**Reply by cobratbq** [reply]
I just reported issue #1861 which is an issue specifically for DNS-over-HTTPS (DoH) resolvers. I started investigating due to the same NOTICE message that you mention above. Are you using _DoH resolvers_ in your configuration?

For me, this error is already present in 2.0.45, and possibly earlier.

**Reply by MojtabaFaal** [reply]
@jedisct1

**Reply by cobratbq** [reply]
Bug #1861 is resolved now. @MojtabaFaal if you were using DoH resolvers, then I'm pretty confident that this is the same issue.
